saleor/product/models.py

```diff
@@ -68,6 +69,7 @@ class ProductType(models.Model):
     variant_attributes = models.ManyToManyField(
         'ProductAttribute', related_name='product_variant_types', blank=True)
     is_shipping_required = models.BooleanField(default=False)
+    tax_rate = models.CharField(max_length=128, default='standard', blank=True)
```

Comment:
Since we use the `standard` value in few places already, we could extract it to a constant.

Reply:
This is exactly what I was planning to do 🙂